.azure/gpu-tests-lite.yml

```diff
@@ -0,0 +1,126 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+  tags:
+    include:
+      - '*'
+  branches:
+    include:
+      - "master"
+      - "release/*"
+      - "refs/tags/*"
+
+pr:
+  - "master"
+  - "release/*"
+
+variables:
+  - name: continue
+    value: '1'
+
+jobs:
+  - job: testing
+    strategy:
+      matrix:
+        'PyTorch - stable':
+          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.12-cuda11.6.1"
+    # how long to run the job before automatically cancelling
+    timeoutInMinutes: "80"
```

Comment:
The timeout should be adjusted, I expect it will take considerably less.

Can be done in a follow-up but I'd add a TODO(lite)

Reply:
Okay, reduced all timeouts in that file